site/pages/[locale]/sign-message.js

```diff
@@ -63,48 +66,45 @@ const SignMessageForm = function () {
   }
 
   return (
-    <>
-      <div className="flex flex-wrap justify-center mt-10 mx-auto w-full max-w-lg space-y-4">
-        <textarea
-          className="w-full border-2"
-          placeholder={t('message-placeholder')}
-          rows={5}
-          title={`${t('message')}:`}
-          {...messageInput}
-        />
-      </div>
-      <div className="mt-7.5 flex justify-center">
-        <Button {...signButton}>{t('sign').toUpperCase()}</Button>
-      </div>
+    <UtilFormBox title={t('sign-message')}>
+      <TextArea
+        className="mb-"
```

Comment:
`mb-` is missing a value at the end

Reply:
removed here: https://github.com/hemilabs/pure.finance/pull/2/commits/cacff18224bff82f9fb329376126dffff2239d91